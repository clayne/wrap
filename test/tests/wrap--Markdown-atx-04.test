wrap | /dev/null | -u | md-atx-04.md | 0
