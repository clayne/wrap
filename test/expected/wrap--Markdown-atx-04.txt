#1 This is not an ATX header.  This is ordinary text.
